tests/testthat/test_HUCTS_aggregator.R

```diff
@@ -0,0 +1,7 @@
+context("Test HUC Time Series Aggregator")
+
+test_that("test package functions", {
+  load(system.file("extdata","HUCTS_aggregator_test.rda",package="HUCAgg"))
+  outData2<-HUCTS_aggregator(upstream_size,fromHUC,huc12_areaDF,huc12agg_areaDF,dataF)
+  expect_equal(outData,outData2)
```

Comment:
Yes. A number of variables are loaded, I've noted it. In the future I'll try and be more explicit about what's getting 'loaded' or I'll use the saveRDS pattern.